specs/core/0_beacon-chain.md

```diff
@@ -810,7 +819,7 @@ def get_beacon_proposer_index(state: BeaconState) -> ValidatorIndex:
 ### `verify_merkle_branch`
 
 ```python
-def verify_merkle_branch(leaf: Hash, proof: List[Hash], depth: int, index: int, root: Hash) -> bool:
+def verify_merkle_branch(leaf: Hash, proof: Tuple[Hash, ...], depth: int, index: int, root: Hash) -> bool:
```

Comment:
This is especially a decrease in understandability imo. It implies to a casual reader that it's a tuple where only the first item needs to be a hash.

Reply:
Fixed, it's `Sequence` now, see explanation below.